example/src/screens/timelineCalendar.js

```diff
@@ -0,0 +1,312 @@
+// }
+
+const EVENTS = [
+  {
+    start: '2017-09-06 22:30:00',
+    end: '2017-09-06 23:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 00:30:00',
+    end: '2017-09-07 01:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 01:30:00',
+    end: '2017-09-07 02:20:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 04:10:00',
+    end: '2017-09-07 04:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 01:05:00',
+    end: '2017-09-07 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-07 14:30:00',
+    end: '2017-09-07 16:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 01:20:00',
+    end: '2017-09-08 02:20:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 04:10:00',
+    end: '2017-09-08 04:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 00:45:00',
+    end: '2017-09-08 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-08 11:30:00',
+    end: '2017-09-08 12:30:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 01:30:00',
+    end: '2017-09-09 02:00:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 03:10:00',
+    end: '2017-09-09 03:40:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-09 00:10:00',
+    end: '2017-09-09 01:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+  {
+    start: '2017-09-10 12:10:00',
+    end: '2017-09-10 13:45:00',
+    title: 'Dr. Mariana Joseph',
+    summary: '3412 Piedmont Rd NE, GA 3032',
+  },
+];
+
+export default class ExpandableCalendarScreen extends Component {
+  onDateChanged = date => {
+    // console.warn('ExpandableCalendarScreen onDateChanged: ', date, updateSource);
+    // fetch and set data for date + week ahead
+    console.log(date);
+  };
+
+  onMonthChange = (/* month, updateSource */) => {
+    // console.warn('ExpandableCalendarScreen onMonthChange: ', month, updateSource);
+  };
+
+  buttonPressed() {
+    Alert.alert('show more');
+  }
+
+  itemPressed(id) {
+    Alert.alert(id);
+  }
+
+  renderEmptyItem() {
+    return (
+      <View style={styles.emptyItem}>
+        <Text style={styles.emptyItemText}>No Events Planned</Text>
+      </View>
+    );
+  }
+
+  renderItem = ({item}) => {
```

Comment:
If you don't use it, remove it.

Reply:
Done!

Reply:
I think you are talking about Alert here.

Reply:
I think what @Inbal-Tish meant is that you are writing empty functions like `buttonPressed` and `itemPressed` which i also agree that it can be removed.
If it doesn't do anything useful, we don't need it. the average developer knows that he can use this `onPress` features of React Native.

Also, I see that you didn't use `renderItem` at all. why do we need it?